common/url_constants.cc

```diff
@@ -0,0 +1,316 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "chrome/common/url_constants.h"
+
+namespace chrome {
+
+const char kAutomaticSettingsResetLearnMoreURL[] =
```

Comment:
can we rewrite/redirect these urls instead of patching them? We have done that in other cases.

Reply:
Sorry, I'm not sure if I understood your comment correctly, I did copy the file into src/brave to rewrite these learn more urls instead of patching?